src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection-overlay/main.js

```diff
@@ -12,6 +12,8 @@
  *
  * @class MediaSelection
  * @constructor
+ *
+ * @deprecated use media-selection/overlay@sulumedia instead
```

Comment:
Has the interface of the new media selection overlay significantly changed? Is it worth keeping  the old file?